docs/src/main/paradox/release-notes/2.0.x.md

```diff
@@ -19,6 +19,29 @@ All Alpakka modules are available for Scala 2.13 and all Alpakka modules are com
 
 **Alpakka 2.0 updates dependencies for most connectors.**
 
+## 2.0.0-RC1
+
+Released 2020-03-20
+
+## Notable changes since 2.0.0-M3
+
+**Alpakka Cassandra is completely rewritten and now based on the Cassandra Java driver version 4.6. It now becomes the base for [Akka Persistence Cassandra](https://doc.akka.io/docs/akka-persistence-cassandra/1.0/).**
+
+- Cassandra: rewrite on top of Java driver 4.6.0 [#2182](https://github.com/akka/alpakka/issues/2182) by [@ennru](https://github.com/ennru)
+
+**The MQTT Streaming connector does only support Akka 2.6 now.**
```

Comment:
(it might have been helpful to mention why: to avoid relying on Akka API's that were still ApiMayChange in 2.5 - but no big deal :) )